stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -1721,6 +1725,20 @@ extension ${Self} {
 }
 
 % if bits == 80:
+#else
+
+${SelfDocComment}
+@_fixed_layout
+@available(*, unavailable, message: "available only #if !os(Windows) && (arch(i386) || arch(x86_64))")
```

Comment:
I think we can probably make the error message a little more approachable; something like "Float80 is only available on non-Windows x86 targets." What do you think?

Reply:
I really had no idea of how `(arch(i386) || arch(x86_64))` would map to reader-friendly text, so I just put the code one would need to verify Float80 was actually available.
Putting your suggestion in.